app/assets/javascripts/admin/addon/components/admin-config-area-cards/about/your-organization.gjs

```diff
@@ -49,12 +95,21 @@ export default class AdminConfigAreasAboutYourOrganization extends Component {
       <p class="admin-config-area-card__additional-help">
         {{i18n "admin.config_areas.about.city_for_disputes_help"}}
       </p>
-      <input type="text" />
+      <input
+        {{on "input" this.onCityForDisputesChange}}
+        type="text"
+        value={{this.cityForDisputes}}
+      />
     </div>
     <DButton
       @label="admin.config_areas.about.update"
       @action={{this.save}}
-      class="btn-primary"
+      class="btn-primary save-card"
```

Comment:
Let's make this more specific for config areas, should be `admin-config-area-card__btn-save`